pyrene/src/components/Accordion/Accordion.tsx

```diff
@@ -30,6 +30,5 @@ const Accordion: React.FC<AccordionProps> = ({
   </div>
 );
 
-Accordion.displayName = 'Accordion';
```

Comment:
Does this work with kitchensink? I think we used to read `displayName` for the component name there. If it's compatible, all good. If not, we might need minor changes there.

Generally though, we should rather tweak kitchensink to support a cool developer experience in pyrene than the other way round (i.e., require anti-patterns in pyrene just because of kitchensink).

Reply:
I'm a bit confused which PR is which, and can't find your comment with the `compareLocale` issue.
However, I am pretty sure that it is related to this comment. Kitchensink currently takes the component name from `displayName` if you remove that, the name has to be taken from somewhere else.